### PR TITLE
Fix env naming convention

### LINKED ISSUE #113: Fix env varibale naming
### Issue Description

Currently, the TokenExpiration and Max_Url_Count is not aligned with the previous naming of env so we need to change the naming convention of it

### Expected Behavior

env naming should be uniform

### Current Behavior

env naming should not be uniform

### Screenshots

[If applicable, add screenshots or images that help illustrate the issue. You can use the following Markdown syntax to embed images:]

### Reproducibility

 - [ ] This issue is reproducible
-  [ ] This issue is not reproducible

### Steps to Reproduce
1. [Step 1]
2. [Step 2]

### Severity/Priority
 - [ ] Critical
 - [ ] High
 - [ ] Medium
 - [ ] Low

### Additional Information

[Include any additional context, logs, or details that might be helpful in addressing the issue.]

### Checklist

- [ ] I have read and followed the project's code of conduct.
- [ ] I have searched for similar issues before creating this one.
- [ ] I have provided all the necessary information to understand and reproduce the issue.
- [ ] I am willing to contribute to the resolution of this issue.

---

Thank you for contributing to our project! We appreciate your help in improving it.

### PR DESCRIPTION
**Pull request by vinit717**
## Issue Ticket Number

-  closes #113 

## Description

In this PR env naming convention is been fixed to align with previous implementation

**Breaking Changes**

-   [ ] Yes
-   [x] No

_If your feature introduces breaking changes or if something is missing, please mention the related issue tickets._

**Development Tested?**

-   [x] Yes
-   [ ] No

_Confirm whether the changes have been tested locally during development._

**Tested in Staging?**

-   [x] Yes
-   [ ] No

_Indicate whether the changes have been tested in the staging environment for dev to main._

**Database Changes**

-   [ ] Yes
-   [x] No

_Indicate whether the changes include modifications to the database._

### Screenshots

Attach any relevant screenshots, such as test coverage reports, before and after images, or other visual aids.

## Additional Notes

Include any additional notes, considerations, or explanations that might be helpful for reviewers.

### CODE REVIEW

File: environments/test.env [hunk 1]
```diff
@@ -14,7 +14,7 @@ GOOGLE_CLIENT_ID="google-client-id"
 GOOGLE_CLIENT_SECRET="google-client-secret"
 GOOGLE_REDIRECT_URL="http://localhost:8000/v1/auth/google/callback"
 
-Max_Url_Count=10
-TokenExpiration=3600
+USER_MAX_URL_COUNT=10
```
Comment (prakashchoudhary07) [inline environments/test.env:17]: NIT: this could have been USER_MAX_URL_LIMIT

**Review by prakashchoudhary07 — APPROVED** [review]

File: environments/test.env (hunk 1 above)
Comment (vinit717) [reply environments/test.env:17]: We can take this later as USER_MAX_URL_COUNT is already added to the env and need to change in another PR also

Comment (prakashchoudhary07) [reply environments/test.env:17]: Then can you please create a separate issue for it and past the link here?

Comment (vinit717) [reply environments/test.env:17]: https://github.com/Real-Dev-Squad/tiny-site-backend/issues/117

**Review by Achintya-Chatterjee — APPROVED** [review]

**Pull request merged**